lib/db.js

```diff
@@ -540,52 +540,61 @@ module.exports = (
     )
   }
 
-  DB.prototype.updateSessionToken = function (token, ip, geo) {
-    log.trace({ op: 'DB.updateSessionToken', uid: token && token.uid })
+  DB.prototype.updateSessionToken = function (token, geo) {
```

Comment:
The `ip` argument wasn't being used any more so I took the opportunity to get rid of it.